src/j1cli.js

```diff
@@ -336,17 +338,42 @@ async function provisionRulePackAlerts(j1Client, rules, defaultSettings) {
   for (const r of rules) {
     if (r.instance) {
       const update = r.instance.id !== undefined;
-      promises.push(j1Client.mutateAlertRule(rule, update));
+      promises.push(j1Client.mutateAlertRule(r, update));
     }
     else {
-      const instance = {
-        name: r.name,
-        description: r.description,
-        question: {
-          queries: r.queries
-        },
-        ...defaultSettings
-      }
+      const instance = r.alertLevel ? 
```

Comment:
I think this could be simplified so that you don't have to specify all the different rule properties. Something like this should work:

```js
      const instance = {
        name: r.name,
        description: r.description,
        question: {
          queries: r.queries
        },
        ...defaultSettings,
        operations: r.alertLevel ? [
          {
            when: defaultSettings.operations[0].when,
            actions: [
              {
                type: "SET_PROPERTY",
                targetProperty: "alertLevel",
                targetValue: r.alertLevel
              },
              {
                type: "CREATE_ALERT"
              }
            ]
          }
        ] : defaultSettings.operations
      }
      promises.push(j1Client.mutateAlertRule({instance}, false))
```